notebooks/population/grid_pop_experiment.py

```diff
@@ -0,0 +1,1218 @@
+# %%
```

Comment:
I didn't review this notebook, just tried to run it. I've added comments where I got errors.